embedded/store/txpool.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2022 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package store
+
+import (
+	"sync"
+
+	"github.com/codenotary/immudb/embedded/htree"
+)
+
+type txPoolOptions struct {
+	PoolSize     int
```

Comment:
options struct may be public? or attributes private?

Reply:
Made members private in 8d479635